api/src/legacy_soap_api/legacy_soap_api_config.py

```diff
@@ -141,7 +141,13 @@ class SOAPOperationConfig:
             response_operation_name="GetApplicationZipResponse",
             is_mtom=True,
             privileges={Privilege.LEGACY_AGENCY_GRANT_RETRIEVER},
-        )
+        ),
+        "GetSubmissionListExpandedRequest": SOAPOperationConfig(
+            request_operation_name="GetSubmissionListExpandedRequest",
+            response_operation_name="GetSubmissionListExpandedResponse",
+            is_mtom=True,
```

Comment:
We may want to turn this off for now till we get the syncing of the two responses going. Right now it should be dererring to the simpler response, which may not be the behavior we want.